spec/models/tag_spec.rb

```diff
@@ -496,6 +519,8 @@
 
         xit "should find all works that would need to be reindexed" do
           # get all the work ids that it would queue
+          User.current_user = create(:admin)
```

Comment:
Why do we need to be an admin here? Also this test is pending anyway.

Reply:
Utterly no idea -- I just knew that we didn't need to be an admin in the scenario with the failure, so I moved the change and checked that the tests passed. I've now removed it entirely as I have no idea why it was originally there. Yay, code from two years ago.